Loan_Default_Classification_Code_Review_V1.py

```diff
@@ -450,33 +456,13 @@ def histogram_alt(feature):
     #msg = "%s: %f" % (name, roc_auc_score)
 
 
-# In[41]:
-
-
-# prepare configuration for cross validation test harness
-seed = 7
-
-# Compare accuracy score among different models
-results = []
-names = []
-scoring = 'accuracy'
-for name, model in models:
-	kfold = model_selection.KFold(n_splits=10, random_state=seed)
-	cv_results = model_selection.cross_val_score(model, X_train, y_train, cv=kfold, scoring=scoring)
-	results.append(cv_results)
-	names.append(name)
-	msg = "%s: %f (%f)" % (name, cv_results.mean(), cv_results.std())
-	print(msg)
-    
-# boxplot algorithm comparison
-fig = plt.figure()
-fig.suptitle('Algorithm Accuracy Comparison')
-ax = fig.add_subplot(111)
-plt.boxplot(results)
-ax.set_xticklabels(names)
-plt.show()
-
-
+"""
```

Comment:
I added some comments in the body of the file, but remove them as you address them. In future iterations this interaction will be cleaner because you can start making PRs and then I can comment on the PRs before merging.